upgrade/upgrade-provider.go

```diff
@@ -27,7 +28,15 @@ func UpgradeProvider(ctx Context, name string) error {
 	var targetBridgeVersion string
 	var upgradeTargets UpstreamVersions
 	var goMod *GoMod
-	upstreamProviderName := strings.TrimPrefix(name, "pulumi-")
+
+	tok := strings.Split(name, "/")
+	if len(tok) != 2 {
+		return errors.New("argument must be provided as {org}/{repo}")
+	}
+	repo.org = tok[0]
+	repo.name = tok[1]
+
+	upstreamProviderName := strings.TrimPrefix(repo.name, "pulumi-")
```

Comment:
Should we verify that `repo.name` starts with `pulumi-`?

Reply:
I should have specified that this should be replaced by https://github.com/pulumi/upgrade-provider/pull/61. We shouldn't assume the upstream is prefixed with `pulumi`, it should be explicitly set in the config/ passed in with `--upstream-provider-name`